packages/expo-speech/src/Speech/ExponentSpeech.web.ts

```diff
@@ -5,6 +5,23 @@ import { SpeechOptions, WebVoice, VoiceQuality } from './Speech.types';
 //https://developer.mozilla.org/en-US/docs/Web/API/SpeechSynthesisUtterance/text
 const MAX_SPEECH_INPUT_LENGTH = 32767;
 
+async function getVoices(): Promise<SpeechSynthesisVoice[]> {
+  return new Promise<SpeechSynthesisVoice[]>((resolve) => {
+    const voices = window.speechSynthesis.getVoices();
+
+    if (voices.length > 0) {
+      resolve(voices);
+      return;
+    }
+
+    // when a page loads it takes some amount of time to populate the voices list
+    window.speechSynthesis.onvoiceschanged = function () {
```

Comment:
Let's add this StackOverflow question for reference here, shall we? 🤔 
```suggestion
    // when a page loads it takes some amount of time to populate the voices list
    // see https://stackoverflow.com/a/52005323/4337317
    window.speechSynthesis.onvoiceschanged = function () {
```
Applying this suggestion would require rebuilding the `build` directory 😞